pkg/imgpkg/bundle/bundle.go

```diff
@@ -157,11 +182,17 @@ func (o *Bundle) pull(baseOutputPath string, ui goui.UI, pullNestedBundles bool,
 				continue
 			}
 
-			subBundle := NewBundle(image.Image, o.imgRetriever)
-			isBundle, err := subBundle.IsBundle()
-			if err != nil {
-				return err
+			var isBundle, ok bool
+			var subBundle *Bundle
```

Comment:
no need for this var, could just use := below

Reply:
The usage of isBundle is outside the scope of the if statement. Hence declaring the variable there.

Reply:
isBundle yeah, but subBundle could just use :=